traitcuration/traits/datasources/ols.py

```diff
@@ -0,0 +1,41 @@
+"""
+This module contains functions to retrieve info about terms from OLS, as well as helper functions such as
+ontology_id extraction
+"""
+import requests
+
+
+BASE_URL = "https://www.ebi.ac.uk/ols/api/"
+
+
+def make_ols_query(term_iri, ontology_id):
+    """
+    Takes in a term iri (or IRI as referenced in OLS documentation) and the ontology id to search against. Returns a
+    dictionary for that term with the fields 'label', 'obo_id' which is the CURIE and 'is_obsolete' as True or False.
+    """
+    results = requests.get(f"{BASE_URL}ontologies/{ontology_id}/terms?iri={term_iri}").json()
+    print(results)
+    if 'error' in results:
+        print
+        return None
+    term_info = results["_embedded"]["terms"][0]
+    term_curie = term_info["obo_id"]  # E.g. EFO:0000400
+    term_label = term_info["label"]  # E.g. Diabetes mellitus
+    term_is_obsolete = term_info["is_obsolete"]  # True or False value on whether the term is obsolete or not
+    info_dict = {"curie": term_curie, "label": term_label, "is_obsolete": term_is_obsolete}
+    print(info_dict)
+    print()
```

Comment:
Not a necessary change (at least for now) but a suggestion: given multiple logging commands (some of which will make it into production, some required only for development), it would be slightly easier to use the `logging` module and to define specific calls as "error", "info", "debug" etc.

Example of using the logging module: https://github.com/EBIvariation/vep-mapping-pipeline/blob/master/vep_mapping_pipeline/consequence_mapping.py (Ctrl+F for "logging" and then "logger")

Reply:
Regarding the error handling, just to clarify that I am proceeding as I should, I am first checking if I get a 404, in which case I return `None` as the result of the query. In any other case, I am using `response.raise_for_status()` to raise an Exception. Should I retry the query in case of unexpected HTTP errors or just for network errors?

Reply:
I've replied in the thread which we already have for error handling above: https://github.com/EBIvariation/trait-curation/pull/49#discussion_r448694485. The GitHub interface is annoying in that, when some comment was made during previous review iterations, it will duplicate it for the newest iteration, but won't allow to actually comment